server/utils/utils.ts

```diff
@@ -458,3 +459,64 @@ export const neurodiversityEnabled = (agencyId: string): boolean => {
   const isEnabled = neurodiversityEnabledPrisons?.includes(agencyId)
   return isEnabled
 }
+
+export const stripAgencyPrefix = (location: string, agency: string): string => {
+  const parts = location && location.split('-')
+  if (parts && parts.length > 0) {
+    const index = parts.findIndex(p => p === agency)
+    if (index >= 0) {
+      return location.substring(parts[index].length + 1, location.length)
+    }
+  }
+
+  return null
+}
+
+export const formatLocation = (locationName: string): string => {
+  if (!locationName) return undefined
+  if (locationName.includes('RECP')) return 'Reception'
+  if (locationName.includes('CSWAP')) return 'No cell allocated'
+  if (locationName.includes('COURT')) return 'Court'
+  return locationName
+}
+
+export const isTemporaryLocation = (locationName: string): boolean => {
+  if (!locationName) return false
+  if (locationName.endsWith('RECP')) return true
+  if (locationName.endsWith('CSWAP')) return true
+  if (locationName.endsWith('COURT')) return true
+  if (locationName.endsWith('TAP')) return true
+  return false
+}
+
+export const formatTimestampToDate = (timestamp: moment.MomentInput, outputFormat = 'DD/MM/YYYY'): moment.MomentInput =>
```

Comment:
Should be using `date-fns` rather than `moment` - also, `dateHelpers.ts` contains some useful functions already, and would be nice to group all date related functions in the same place.